Dockerfile

```diff
@@ -24,4 +24,5 @@ COPY --from=builder /build/fakepasswd /etc/passwd
 
 EXPOSE 8080
 
-ENTRYPOINT ["/go-test-bench"]
\ No newline at end of file
+# default listen address of localhost:8080 does not work
```

Comment:
I tried 127.0.0.1 and that failed as well, I think because the container doesn't have an interface with that address. I'm open to rewording, but the problem is bigger than just localhost missing from /etc/hosts :)

Reply:
That's disappointing. Thanks for verifying. I really wish we had a better understanding of what's happening here but it isn't worth holding up the PR for.